docs/home/whats-new-2-15.md

```diff
@@ -1,94 +1,103 @@
+<!-- RUN-10639/RUN-11389 - Researcher Service Refactoring -->
+* Improved configuration for authentication in the cluster. Configuring the Kubernetes cluster API server for authentication is no longer needed. This applies to environments where end users are using the UI only and not the CLI.
 
-<!-- RUN-10802 Quota management dashboard post-release -->
+<!-- RUN-12505/RUN-12506 - Support Kubeflow notebooks for scheduling/orchestration -->
+* Improved support for KubeFlow Notebooks. Now Run:ai supports scheduling of Kubeflow notebook CRDs with fractional GPUs. Kubeflow notebooks are identified automatically and use a special icon in the *Jobs* UI.
 
-<!-- RUN-10862/RUN-10863 Department as a workspace asset creation scope - phase 2-->
+<!-- RUN-10251/RUN-10252 - Block over-subscription of quota by Projects/Departments- -->
+* Improved quota settings by providing the ability to block over-subscription of quota by *Projects* or *Departments*. For more information, see [Over quota blocking](../Researcher/scheduling/the-runai-scheduler.md#limit-quota-over-or-under-subscription).
 
-<!-- RUN-11186/RUN-11241 Submitting an MPI/PT/TF/XGBoost distributed training from UI - 1st form page -->
+<!-- RUN-10404/RUN-11747 Submit distributed training
+* RUN-11194/RUN-11239 All changes done in the UI for distributed training are hidden behind feature flag 
+* RUN-11186/RUN-11241 Submitting an MPI/PT/TF/XGBoost distributed training from UI - 1st form page 
+* RUN-11206/RUN-11242 Submitting distributed training from UI - 2nd form page
+* RUN-11219/RUN-11602 Submitting an MPI distributed training from UI - 3rd form page
+* RUN-11231/RUN-11240 Environment for distributed training
+-->
+* Added support for distributed training in the following:
 
-<!-- RUN-11194/RUN-11239 All changes done in the UI for distributed training are hidden behind feature flag -->
+      * Trainings form
+      * Environments form
+  
+    You can select `single` or `multi-node (distributed)` training. When configuring distributed training, you will need to select a framework from the list. For *Trainings* configuration, see [Adding trainings](../Researcher/user-interface/trainings.md#adding-trainings). See your Run:ai representative to enable this feature. For *Environments* configuration, see [Creating an Environment](../Researcher/user-interface/workspaces/create/create-env.md#creating-a-new-environment).
 
-<!-- RUN-11206/RUN-11242 Submitting distributed training from UI - 2nd form page-->
+<!-- ADDLINK RUN-10411/RUN-11390 Support self-signed certificates-->
+* Run:ai can be installed in an isolated network. In this air-gapped configuration, the organization will not be using an established root certificate authority{target=_blank}. Instead, the organization creates a local certificate which serves as the root certificate for the organization. The certificate is installed in all browsers within the organization. For more information, see [link here](##org-cert.md).
```

Comment:
@jasonnovichRunAI looks good except for "This allows.." because it sounds like "This" refers to the sentence before (which just describes the current state), but actually it refers to the new feature. Maybe we can say "Run:ai now allows..." instead?